.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea folder
-out/ folder
-.iml files
```

Comment:
For specific file types use pattern matching.

First we will consider the first version of your .gitignore.

Example from: https://stackoverflow.com/a/33190653/6541172
```
* exclude every file in the repository.
*.iml exclude every .iml file in the repository.
!*.html allow all html files.
!images/*.* consider all types of file in images folder.
```